translations/translation_hu.xml

```diff
@@ -385,20 +385,20 @@
 		<text name="COURSEPLAY_RELOAD_COURSE_LIST" 							text="Útvonallista újratöltése" />
 		<text name="COURSEPLAY_COULDNT_GENERATE_COURSE" 					text="Útvonalgenerálás sikertelen! Próbálj változtatni a beállításokon vagy állj arréb a járművel!"/>
 		
-		<text name="COURSEPLAY_FillType_BaleTwine" 							text="bale twine" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
-		<text name="COURSEPLAY_FillType_BaleNet" 							text="bale net" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
-		<text name="COURSEPLAY_FillType_WrappingFoil" 						text="wrapping foil" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
-		<text name="COURSEPLAY_FillType_Molasses"							text="molasses" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
+		<text name="COURSEPLAY_FillType_BaleTwine" 							text="bála zsineg" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
+		<text name="COURSEPLAY_FillType_BaleNet" 							text="bála háló" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
+		<text name="COURSEPLAY_FillType_WrappingFoil" 						text="csomagoló fólia" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
+		<text name="COURSEPLAY_FillType_Molasses"							text="melasz" /> <!-- TODO TRANSLATION  supply fill types for straw harvest addon-->
 
 		<text name="COURSEPLAY_AUTODRIVE_MODE"				    			text="AutoDrive" />
-		<text name="COURSEPLAY_AUTODRIVE_NO_AUTODRIVE"				    	text="Not installed" />
-		<text name="COURSEPLAY_AUTODRIVE_DONT_USE"				    		text="Don't use" />
-		<text name="COURSEPLAY_AUTODRIVE_UNLOAD_OR_REFILL"					text="For unload/refill" />
+		<text name="COURSEPLAY_AUTODRIVE_NO_AUTODRIVE"				    	text="Nincs telepítve" />
+		<text name="COURSEPLAY_AUTODRIVE_DONT_USE"				    		text="Ne használd" />
+		<text name="COURSEPLAY_AUTODRIVE_UNLOAD_OR_REFILL"					text="For unload/orefill" />
```

Comment:
This one isn't translated (which is not a problem but now it has a typo in it)

Reply:
In line 396 that was a misspelling, but the rest is correct. Just simply ignore that line.

Reply:
Ne haragudj de nem értem, azt akarod, hogy "orefill" legyen ott?

Reply:
No. "orefill" is just a misspelling. I accidentally pressed "o" when the cursor was there and I recognised it only after I clicked to submit. Just leave it in its original state ("refill"). Where are you from btw?

Reply:
I can't just leave it there, that's not how git works, either I merge everything or nothing.

It would be a lot easier for me if you fixed it and pushed another commit, that way I only have to click here once to merge, that's why I requested a change.

If you don't fix it and I merge it as it is then I have to edit the file, commit it and push it.

I'm from Hungary but living in the US.

Reply:
Asszem most már jó lesz. Király cucc ez a courseplay, csak azt sajnálom, hogy a kombájnkövetésnél (mode2) a kombájn és a traktor/kamion nem tudnak szépen együtt dolgozni. Kíváncsian figyelem a fejleményeket és remélem, egy napon sikerül igazán jól összerakni ezt a munkatípust. Jó éjt!